tests/path.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+
+'''
+Since tweetme.py is now present in parent directory we need to add appropriate path in order to
```

Comment:
Not sure why this is needed. I still need to test this code locally once. But till the time you can look into other things.

Reply:
In order to `import tweetme` which is not in the parent directory where all the tests files are, we had to add the location of the folder where `tweetme` is in order to access it as a module. This is where `path.py` comes it.